sympy/simplify/cse_main.py

```diff
@@ -657,6 +726,8 @@ def cse(exprs, symbols=None, optimizations=None, postprocess=None,
         concern, use the setting order='none'.
     ignore : iterable of Symbols
         Substitutions containing any Symbol from ``ignore`` will be ignored.
+    as_list : bool, (default True)
+        Returns expression in list or else with same type as input (when False).
```

Comment:
From this explanation it isn't clear what this does. An example would help.

Reply:
By default, if we do not call 'as_list = False', then no test bench works and lambdify will systematically return a list. Which would profoundly change the nature of 'lambdify'. However, by calling 'as_list = False', the behavior of 'lambdify' remains the same regardless of the value of the 'cse' flag.

Reply:
There is an example in the docstring.